decidim-core/app/forms/url_validator.rb

```diff
@@ -6,7 +6,7 @@
 #
 class UrlValidator < ActiveModel::EachValidator
   def validate_each(record, attribute, value)
-    record.errors.add attribute, (options[:message] || "must be a valid URL") unless url_valid?(value)
+    record.errors.add attribute, :url_format, **options unless url_valid?(value)
```

Comment:
A default message should be added as well, in the decidim-core, most likely a key like : 
```
en:
  errors:
    messages:
      url_format: 
```
or 
```
en:
  activerecord:
    errors:
      messages:
        url_format: 
```
You can play with it, by commenting the error message override from the above file, then attempt to customize the generic message that is being displayed. 

The I18n field translation is checking to see if there is any specific error attached on the field, model otherwise it would us the default message. In our case we are missing the default message.

Reply:
Thanks, I have updated the default message to a slightly different format in EN